src/fiatExchanges/types.ts

```diff
@@ -7,3 +10,16 @@ export enum SelectProviderExchangesLink {
   ExternalExchangesScreen = 'ExternalExchangesScreen',
   ExchangeQRScreen = 'ExchangeQRScreen',
 }
+
+export interface ProviderSelectionAnalyticsData {
```

Comment:
It seems to me that `flow` would be another natural value to put in this set. Any reason why its left out?

Reply:
We don't need this for coinbase pay as it's cash in only, but I agree its good to include. I tried doing this, but it turns out to be a big refactor (causes cyclic imports on types, CICOFlow is defined in utils, ProviderSelectionAnalyticsData is in types). I think we can defer this until coinbase pay events need flow as well.